docs/defining_new_slo.md

```diff
@@ -0,0 +1,191 @@
+SLO should be measured over some longer time period. Most common would be a month meaning "We want to have a 98% availability over the last month".
+This purely depends on the service nature and possibly on the users and what they expect.
+
+## Documenting SLO
+It's good practice to document how the SLO is measured in a human-readable form, so the product owner understands it.
+Remember that each change in the SLO computation should be communicated, understood and updated there.
+
+> Example:
+> Our availability SLO is 99.5% of successful events over 4 weeks.
+> Failed event is an HTTP request from the user resulting with status code 5xx.
+> All requests resulting with 4xx status codes are not counted into the total amount of events since those are user-side issues.
+> Also all requests made by our external monitoring are excluded from the whole SLO computation.
+
+
+# 3. Implementing SLO
+The SRE workbook contains examples how to implement SLO using Prometheus metrics.
+Initially we went this path and in some cases it can be the easiest way to start and eventually sufficient.
+But in some cases you can encounter issues that are hard to overcome and that was the reason we created the slo-exporter.
+If you want to read more about it take a look at [this series of blog posts we wrote](https://medium.com/@sklik.devops/our-journey-towards-slo-based-alerting-bd8bbe23c1d6).
+
+Ideally everyone in company should use the same approach how to compute or at least visualize SLOs. 
+In case slo-exporter is too much of a complexity for your trivial SLO computation,
+we advise you to at least try to stick to the metric naming used by slo-exporter and possibly using
+at least its alerting rules and dashboards.
+
+## Using Prometheus only
+This can be easy to implement, and you can find number of upstream
+projects that can help you with that such as [this SLO generator](https://promtools.dev/alerts/errors), 
+[Sloth](https://sloth.dev/) or [the Pyrra project](https://github.com/pyrra-dev/pyrra) having even a nice visualization.
+
+#### Advantages
+- No need for application changes, you can use already existing Prometheus metrics you already collect
+- No need for other infrastructure changes
+
+#### Disadvantages
+- Limited by cardinality because of the Prometheus architecture and purpose
+  (if needed for filtering of events or having more precise thresholds for some events)
+- For latency and quality SLI types need to use histograms and its buckets need to match the thresholds
+- Computing SLO based on high cardinality metrics over 4w can be enormously demanding for the Prometheus cluster
+
+## Using slo-exporter
+Slo exporter was built to solve more complex SLO setups and to unify SLO computation.
+It has support to compute SLO based on querying Prometheus data, tailing logs from file, receiving logs from Envoy proxy, following Kafka topic or reading logs from Elasticsearch.
+
+### 1. Choose source of the events
+#### Querying Prometheus
```

Comment:
Please link this from the prometheus-ingester module's doc for cases when users would skip this doc and go directly for module's documentation

Reply:
done here https://github.com/seznam/slo-exporter/pull/86/commits/8124d42eef4dadd3ca48e00ae52980040e02e07f